### ISSUE TITLE
Transaction date in US format

### ISSUE THREAD

**Issue by thiagofigueiro** [cronoh/nanovault#81]
Can we please have the date format in Account Details in ISO 8601 format?  Following the regional format of the browser language would also be acceptable.

Currently, it seems the date format is hard-coded to `m/d/yy` (US).

**Comment by cronoh** [comment]
@thiagofigueiro Yeah that sounds doable - thanks for the suggestion!  I'll update here when it is implemented.
